rosbag2_storage_default_plugins/test/rosbag2_storage_default_plugins/sqlite/test_sqlite_storage.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include <gmock/gmock.h>
 
+#include <rosbag2_storage/storage_filter.hpp>
+
```

Comment:
style: please adhere to the applied style and include this with `""` below the system headers.

Reply:
59d4cb1